.github/workflows/build.yml

```diff
@@ -421,7 +421,7 @@ jobs:
         if: contains(github.event.pull_request.user.login, 'dependabot') == false
         uses: actions-ecosystem/action-add-labels@v1.1.3
         with:
-          github_token: ${{ steps.keyvault-yaml-secret.outputs.ACTIONS-API-ACCESS-TOKEN }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
It may work without this line since github_token is default: https://github.com/actions-ecosystem/action-add-labels#inputs

Reply:
Removed